app/services/locations/updater.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Locations
+  class Updater
+    YOI_NOMIS_AGENCIES = %w[WYI WNI CKI PRI FMI].freeze
+
+    def self.call
+      Location.update_all(yoi: false)
```

Comment:
maybe best to wrap those in a transaction to avoid having that millisecond where all location YOIs are false 👹  or if there is a failure etc.

Reply:
Okay will do